### PR TITLE
Add Project Gezond scraper (Dutch Website)

### PR DESCRIPTION
**Pull request by NijeboerFrank**
Thanks for this project! I use it a lot on my Mealie instance.

This PR contains a Dutch recipe site that I use sometimes ([Link](https://www.projectgezond.nl/)). Sadly the site did not have a nice way of using the schema nor did the HTML have useful class names, so my implementation might seem a bit hacky. 

Feedback is appreciated to improve the implementation!

### CODE REVIEW

**Comment by jayaddison** [comment]
Thanks @NijeboerFrank for your contribution!  This implementation looks really good to me.

One nitpick I had was about the `category` field (the newline), and it seems like that's also something the unit tests are complaining about at the moment.  Hopefully a quick fixup.

> Sadly the site did not have a nice way of using the schema nor did the HTML have useful class names

That's ok - this library is here to help with situations like that :)

> Feedback is appreciated to improve the implementation!

One more item I noticed: we could replace `.get_text(...)` calls with `.text` attribute accesses.  Let me scour for any other improvements as well.

File: recipe_scrapers/projectgezond.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            ]
+        ).strip()
+
+    def yields(self):
+        return re.search(
+            r"\(([^)]+)",
+            self.soup.find(
+                "h2", string=lambda x: x.startswith("Dit heb je nodig")
+            ).get_text(),
+        ).group(1)
```
Comment (jayaddison) [inline recipe_scrapers/projectgezond.py:42]: Would this match _any_ text content within parentheses inside the `h2`?

(I'm wondering whether non-yields-related information might be scraped unintentionally from some pages by this code)

Comment (NijeboerFrank) [reply recipe_scrapers/projectgezond.py]: I changed the regex slightly in my new commit. I now match everything inside the parentheses after 'Dit heb je nodig '. I checked the website and it seems that that's the place where they always put the yields.

File: tests/test_projectgezond.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def test_title(self):
         self.assertEqual("Boeuf bourguignon", self.harvester_class.title())
 
     def test_category(self):
-        self.assertEqual("Diner\nKerstrecepten", self.harvester_class.category())
+        self.assertEqual(["Diner", "Kerstrecepten"], self.harvester_class.category())
```
Comment (jayaddison) [inline tests/test_projectgezond.py:21]: Ah, this helped to uncover a related problem: calls to `scraper.category()` should return a string type, not a `list` (as here).

We don't have type hints in place to make that clearer, but there is work-in-progress to add those.

Comment (jayaddison) [reply tests/test_projectgezond.py]: The fix looks good, thanks!

File: recipe_scrapers/projectgezond.py (hunk 1 above)
Comment (jayaddison) [reply recipe_scrapers/projectgezond.py]: That change looks good too.  As you suggest, perhaps it's a little bit fragile (may break if & when the site layout/text content changes) -- but those kind of issues can always be resolved over time.

File: recipe_scrapers/projectgezond.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                element.text
+                for element in time_element.children
+                if element.text != "Bereidingstijd:"
+            ]
+        ).strip()
+
+    def yields(self):
+        # Match everything in the h2 with 'Dit heb je nodig'
+        # The text inside the parentheses contains the yield for the ingredients that are listed
+        return re.search(
```
Comment (jayaddison) [inline recipe_scrapers/projectgezond.py:43]: One more thing to keep in mind - hopefully not important in this case, but in general - and sorry if I'm explaining things that you understand already, but it's worth being careful to limit what regular expressions can match on, and/or how much input text they are provided as input.

Just something I repeat (no pun intended) at nearly every available opportunity :smile:

**Comment by jayaddison** [comment]
This looks good to me - thanks again @NijeboerFrank.  I'll plan to merge and release this in the nearish future (possibly not today, but should be within the next few days otherwise).

**Pull request merged**